products/prometheus.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Prometheus
+category: server-app
+permalink: /prometheus
+releasePolicyLink: https://prometheus.io/docs/introduction/release-cycle/
+activeSupportColumn: false
+releaseDateColumn: true
+
+auto:
+-   git: https://github.com/prometheus/prometheus.git
+
+identifiers:
+-   repology: prometheus
+-   purl: pkg:docker/prom/prometheus
+-   purl: pkg:oci/prometheus?repository_url=quay.io/repository/prometheus
+-   purl: pkg:github/prometheus/prometheus
+
+changelogTemplate: https://github.com/prometheus/prometheus/releases/tag/v__LATEST__
+
+# eol(x) = releaseDate(x) + 6m (non-LTS)
```

Comment:
Shouldn't this be 6 weeks, as stated on https://prometheus.io/docs/introduction/release-cycle/ :

> Every 6 weeks, a new Prometheus minor release cycle begins. After those 6 weeks, minor releases generally no longer receive bugfixes. If a user is impacted by a bug in a minor release, they often need to upgrade to the latest Prometheus release.

From what I read I understand : `eol(x) = releaseDate(x+1)`